docs/source/en/tasks/object_detection.md

```diff
@@ -362,7 +363,7 @@ Face to upload your model).
 >>> training_args = TrainingArguments(
 ...     output_dir="detr-resnet-50_finetuned_cppe5",
 ...     per_device_train_batch_size=8,
-...     num_train_epochs=10,
+...     num_train_epochs=100,
```

Comment:
Did you manage to investigate this? 100 epochs is very large. If we train for this many steps do we still see a change in metrics over the first 10 epochs?